src/GraphODataTemplateWriter/.config/TemplateWriterSettings.json

```diff
@@ -2,28 +2,30 @@
     "AvailableLanguages": [ "Android", "ObjC", "CSharp", "Python", "JavaScript" ],
     "TargetLanguage": "CSharp",
     "Plugins": [ ],
-    "PrimaryNamespaceName": "com",
+    "PrimaryNamespaceName": "oneDrive",
     "NamespacePrefix": "com",
     "InitializeCollections": "false",
     "AllowShortActions": "false",
-    "NamespaceOverride": "",
+    "NamespaceOverride": "Microsoft.OneDrive.Sdk",
     "TemplatesDirectory": "../../../../Templates",
     "DefaultFileCasing": "UpperCamel",
     "CustomFlags": [ "python2" ],
     "LicenseHeader": [
         "Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information."
     ],
+    "AdditionalMethodProperties": {
+        "oneDrive.delta": {
+            "CollectionResponse": {
+                "@delta.token": "Edm.String",
```

Comment:
Sure, there could be plenty. This is a delta concept in general, which would apply to the other usages of delta in Graph. There are also some navigation property requests that have required parameters that aren't discoverable at all in metadata. We could follow this pattern for those.

Agreed this is a hack and we should do this using something like annotations. And it would be removed when there's a better answer. Not sure it's on any radar and I don't want to wait 6 months for that to happen before we consolidate all this logic. Fixing the duplication issues and reducing maintenance costs is so valuable. @rgregg, thoughts on feasibility of fixing this via annotations soon?